python/dgl/nn/mxnet/conv/graphconv.py

```diff
@@ -32,10 +32,18 @@ class GraphConv(gluon.Block):
     out_feats : int
         Output feature size; i.e., the number of dimensions of :math:`h_i^{(l+1)}`.
     norm : str, optional
-        How to apply the normalizer. If is `'right'`, divide the aggregated messages
-        by each node's in-degrees, which is equivalent to averaging the received messages.
-        If is `'none'`, no normalization is applied. Default is `'both'`,
-        where the :math:`c_{ij}` in the paper is applied.
+        How to apply the normalizer.  Can be one of the following values:
+
+        * ``right``, to divide the aggregated messages by each node's in-degrees,
+          which is equivalent to averaging the received messages.
+
+        * ``none``, where no normalization is applied.
+
+        * ``both`` (default), where the messages are scaled with :math:`c_{ji}` above, equivalent
+          to symmetric Laplacian normalization.
```

Comment:
The term "Laplacian" here can be confusing, perhaps it's better to say "adjacency normalization".